examples/ses/template.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: ses.aws.crossplane.io/v1alpha1
+kind: Template
+metadata:
+    name: example
+spec:
+    forProvider:
+    region: us-east-2
+    template:
+        templateName: example
+        htmlPart: "<h1>sample html part</h1>"
+        subjectPart: "sample subject part"
+        textPart: "sample text part"
+    providerConfigRef:
+    name: example
```

Comment:
nitpick: if all other files have a final newline, it would be nice to add it to this file and `generator-config.yaml` too.

Reply:
Added newline to the examples.